docs/documentation/release_notes/topics/26_0_0.adoc

```diff
@@ -130,6 +130,10 @@ not recommended at all in production deployments of Keycloak, it is fairly frequ
 of `localhost`. As an alternative to the `_LEGACY` cookies Keycloak now doesn't set the `secure` flag and sets `SameSite=Lax`
 instead of `SameSite=None` when it detects an insecure context is used.
 
+= Hostname v1 feature removed
+
+The deprecated hostname v1 feature was removed. This feature was deprecated in {project_name} 24 and replaced by the hostname v2 theme. If you are still using this feature, you should migrate to the hostname v2 feature.
+
```

Comment:
I'd explicitly mention that the `proxy` option has been finally removed, and users need to migrate to the `proxy-headers`?

Reply:
Yes, I'll add that too.